app/jobs/modsulator_job.rb

```diff
@@ -124,13 +127,13 @@ def generate_original_filename(uploaded_filename)
 
   # Write initial job information to the log file.
   #
-  # @param [File]    log_file  The log file to write to.
-  # @param [String]  username  The login name of the current user.
-  # @param [String]  filename  The name of this job's input file.
-  # @param [String]  note      An optional comment that describes this job.
-  def start_log(log_file, username, filename, note = '')
+  # @param [File] log_file The log file to write to.
+  # @param [user] username The login name of the current user.
```

Comment:
I think this should be `@param [User] user` instead of `@param [user] username`?